ext_conf_template.txt

```diff
@@ -0,0 +1,2 @@
+# cat=modules//; type=string; label= Path to (protected) upload folder. : Example '3:/user_upload/userimport/' where the id (3) has to match the uid of your (protected) storage record.
```

Comment:
I suggest to match the example with the readme example (protected vs user_upload)